src/main/java/uk/gov/hmcts/ccd/domain/service/common/AccessControlServiceImpl.java

```diff
@@ -55,14 +54,17 @@ public AccessControlServiceImpl(final ApplicationParams applicationParams,
     public boolean canAccessCaseTypeWithCriteria(final CaseTypeDefinition caseType,
                                                  final Set<AccessProfile> accessProfiles,
                                                  final Predicate<AccessControlList> criteria) {
-        boolean hasAccess = caseType != null
-            && hasAccessControlList(accessProfiles, criteria, caseType.getAccessControlLists());
+        if (caseType == null) {
+            LOG.debug("Missing caseType");
+            return false;
+        }
 
+        boolean hasAccess = hasAccessControlList(accessProfiles, criteria, caseType.getAccessControlLists());
         if (!hasAccess) {
-            LOG.debug("No relevant case type access for caseType={}, caseTypeACLs={}, userRoles={}",
-                caseType != null ? caseType.getId() : "",
-                caseType != null ? caseType.getAccessControlLists() : Lists.newArrayList(),
-                accessProfiles);
+            LOG.debug(AccessControlService.NO_ROLE_FOR_ACCESS, "caseType",
```

Comment:
jurisdiction Id is not supplied here since each caseType is being called based on the jurisdiction. [See here for implementation](https://github.com/hmcts/ccd-data-store-api/blob/e6d5579f206077c006f9ca7999ffbecca9bc89f9/src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/AuthorisedGetUserProfileOperation.java#L43)